web/docs/auth/migrate-from-11.md

```diff
@@ -2,13 +2,73 @@
 title: Migration from 0.11.X to 0.12.X
```

Comment:
@sodic Should we move this file away from the auth folder into the root?

Reply:
Definitely

Reply:
I moved it to "Getting started", seemed like the best and quickest solution.